site/app/controllers/AuthenticationController.php

```diff
@@ -72,6 +72,10 @@ public function logout() {
      * @return MultiResponse
      */
     public function loginForm($old = null) {
+        if (!is_null($old) && !Utils::startsWith(urldecode($old), $this->core->getConfig()->getBaseUrl())) {
+            $old = null;
+        }
+        header('X-Frame-Options: DENY'); // Stops clickjacking on all pages
```

Comment:
This should be set in `site/public/index.php`, and let's use `SAMEORIGIN` to continue to allow for Barb's original use-case of embedding Submitty in an iframe on her course website. Ideally this would also be a separate PR, as it's a different security thing as the above, regardless of how small a change it is currently, but we can leave it in the same PR here since it's already started.

Reply:
I don't use iframes anymore -- haven't ever since we moved all the menu options to the left sidebar.  I changed my website to mimc the submitty styling (but no page is a mix of domains).  I don't think we should continue to support external iframes.

Reply:
I think another issue with SAMEORIGIN is that a member of RPI (or a hacked account) could use their personal homepage to make a fake Submitty on the same origin